doc/tutorials/objdetect/aruco_board_detection/aruco_board_detection.markdown

```diff
@@ -0,0 +1,194 @@
+easily.
+
+The ```cv::aruco::GridBoard``` class is a specialized class that inherits from the ```cv::aruco::Board``` class and which represents a Board
+with all the markers in the same plane and in a grid layout, as in the following image:
+
+![Image with aruco board](images/gboriginal.jpg)
+
+Concretely, the coordinate system in a Grid Board is positioned in the board plane, centered in the bottom left
+corner of the board and with the Z pointing out, like in the following image (X:red, Y:green, Z:blue):
+
+![Board with axis](images/gbaxis.jpg)
+
+A ```cv::aruco::GridBoard``` object can be defined using the following parameters:
+
+- Number of markers in the X direction.
+- Number of markers in the Y direction.
+- Length of the marker side.
+- Length of the marker separation.
+- The dictionary of the markers.
+- Ids of all the markers (X*Y markers).
+
+This object can be easily created from these parameters using the ```cv::aruco::GridBoard``` constructor:
+
+@snippet samples/cpp/tutorial_code/objectDetection/detect_board.cpp aruco_create_board
+
+- The first and second parameters are the number of markers in the X and Y direction respectively.
+- The third and fourth parameters are the marker length and the marker separation respectively. They can be provided
+in any unit, having in mind that the estimated pose for this board will be measured in the same units (in general, meters are used).
+- Finally, the dictionary of the markers is provided.
+
+So, this board will be composed by 5x7=35 markers. The ids of each of the markers are assigned, by default, in ascending
+order starting on 0, so they will be 0, 1, 2, ..., 34.
+
+After creating a Grid Board, we probably want to print it and use it. A function to generate the image
+of a ```cv::aruco::GridBoard``` is provided in ```cv::aruco::GridBoard::generateImage()```:
+
+@snippet samples/cpp/tutorial_code/objectDetection/create_board.cpp aruco_generate_board_image
+
+
+A full working example of board creation is included in the `samples/cpp/tutorial_code/objectDetection/create_board.cpp`
+
+
+- The first parameter is the size of the output image in pixels. In this case 600x500 pixels. If this is not proportional
+to the board dimensions, it will be centered on the image.
+- ```boardImage```: the output image with the board.
+- The third parameter is the (optional) margin in pixels, so none of the markers are touching the image border.
+In this case the margin is 10.
+- Finally, the size of the marker border, similarly to ```generateImageMarker()``` function. The default value is 1.
+
+The output image will be something like this:
+
+![](images/board.png)
+
+A full working example of board creation is included in the `samples/cpp/tutorial_code/objectDetection/create_board.cpp`.
+
+Note: The samples now take input via commandline via the [OpenCV Commandline Parser](http://docs.opencv.org/trunk/d0/d2e/classcv_1_1CommandLineParser.html#gsc.tab=0). For this file the example parameters will look like
```

Comment:
Please use doxyen/markdown anchor, but not http links.

Reply:
fixed:
![image](https://github.com/opencv/opencv/assets/22337800/51ffc26e-ee25-463a-8230-c84186ea8459)